mlos_bench/mlos_bench/storage/sql/experiment.py

```diff
@@ -93,6 +93,20 @@ def load_config(self, config_id: int) -> Dict[str, Any]:
         with self._engine.connect() as conn:
             return self._get_params(conn, self._schema.config_param, config_id=config_id)
 
+    def load_telemetry(self, trial_id: int) -> List[Tuple[datetime, str, Any]]:
+        with self._engine.connect() as conn:
+            cur_telemetry = conn.execute(
+                self._schema.trial_telemetry.select().where(
+                    self._schema.trial_telemetry.c.exp_id == self._experiment_id,
+                    self._schema.trial_telemetry.c.trial_id == trial_id
+                ).order_by(
+                    self._schema.trial_telemetry.c.ts,
+                    self._schema.trial_telemetry.c.metric_id,
+                )
+            )
+            return [(row.ts, row.metric_id, row.metric_value)
+                    for row in cur_telemetry.fetchall()]
```

Comment:
Seems like a good candidate for a DataFrame instead of a list of tuples.
We lose schema info I suppose though ...

Reply:
yes - but if remember correctly, we've decided not to use pandas anywhere in the Storage or Environment API

Reply:
Ah, could have been and I just forgot.
Am just thinking that with the amount of data for telemetry in particular and possibly wanting to slice and dice it different ways that might be more efficient, but we can tackle that later if it shows up.